pkg/minikube/image/image.go

```diff
@@ -102,18 +102,6 @@ func DigestByGoLib(imgName string) string {
 	return cf.Hex
 }
 
-// Tag returns just the image with the tag
-// eg image:tag@sha256:digest -> image:tag if there is an associated tag
-// if not possible, just return the initial img
-func Tag(img string) string {
```

Comment:
Why this was deleted?

Reply:
Hi @medyagh 

Thanks for your review.

This is for clean code.

I noticed that this `Tag` function is only referred in this [output.Waring](https://github.com/kubernetes/minikube/pull/13787/files/fe7875c01d010418463904c3ae83eeccb4d64218#diff-8eb0087df66277cbdc1a13bd11222d91a100bc0e90013e852264816f4b95ab98L133). 

The full image name including SHA is more clear.
Especially, it may make users confused if the fallback image tag is the same as original one.

e.g.

With `Tag` function:
original one: image:tag
fallback: image:tag

Without `Tag` function:
original one: image:tag@sha256:digest
fallback: image:tag

So it's better to show the full image name including SHA.

The only reference of `Tag` is removed in this PR. In order to make this project simple, I'd like to remove the definition and related test. If we leave it here, it would become harder to delete it next time.

But if you insist, let us keep it in code base.

Let me know what you think

Reply:
@zhan9san thank you for the clarification,

It is still important to  shorten the First one , since Out is meant for the UI and and having long URLS in the UI will break and not look readable.


so I recommend to please keep this